### PR TITLE
Add recent Lincoln events

### PR DESCRIPTION
**Pull request by blainekasten**

### CODE REVIEW

File: _presenters/presenters.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
 	"users": [
+    "1Copenut",
```
Comment (zachleat) [inline _presenters/presenters.json]: Can you fix the indentation here? I think it’s mixing tabs/spaces.

File: dist/0.1.6/global.js [hunk 2]
```diff
@@ -1,203 +1,7 @@
-/* nebraskajs - v0.1.6 - 2015-04-03
+/* nebraskajs - v0.1.6 - 2015-04-23
```
Comment (zachleat) [inline dist/0.1.6/global.js]: I think you may need to pull in master or run an npm install or something here. It’s missing some content.

**Comment by blainekasten** [comment]
Alright. I did that, does it fix those things properly? @zachleat

**Comment by zachleat** [comment]
LGTM!

**Pull request merged**